pages/rules.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>Quizmee | Rules</title>
+</head>
+<body class="bg-grey">
+    <div class="main-container">
+        <h2 class="rule-heading text-white center">RULES</h2>
+        <div class="quiz-category-container center flex">
+            <div class="quiz-container text-white">
+                <ul>
+                <h3><li>Take a chill pill first, it's just a game.</li></h3>
+                <h3><li>Each quiz category has 5 questions.</li></h3>
+                <h3><li>Every correct answer gets you <span class="text-green">+10</span> points.</li></h3>
+                <h3><li>Every wrong answer will cost you <span class="text-yellow">0</span>, <span class="text-orange">-10</span> & <span class="text-red">-20</span> for Easy, Medium & Hard modes respectively.</li></h3>
+                <h3><li>Maximum points are <span class="text-green">50</span>.</li></h3>
```

Comment:
don't use `li` inside a heading tag, if you want to make it big, then you can use a utility class to increase its font size.

Reply:
okay will look into that